package.json

```diff
@@ -21,6 +21,7 @@
     "lint": "tslint \"src/*\" -c tslint.json tsconfig.json",
```

Comment:
I'd just append `&& yarn prettier-check` to the end of this like @ds300 was suggesting.